xonsh/procs/proxies.py

```diff
@@ -283,6 +285,69 @@ def parse_proxy_return(r, stdout, stderr):
     return cmd_result
 
 
+def proxy_zero(f, args, stdin, stdout, stderr, spec, stack):
```

Comment:
this dispatcher can be used where an alias with less number of args can be satisfied. https://github.com/xonsh/xonsh/blob/2d966fb1b56f2399b9753880744645b62133e982/xonsh/cli_utils.py#L369  .